mesheryctl/internal/cli/root/system/check.go

```diff
@@ -40,13 +40,13 @@ func runDockerHealthCheck() error {
 	log.Info("\nDocker \n--------------")
 	//Check whether docker daemon is running or not
 	if err := exec.Command("docker", "ps").Run(); err != nil {
-		log.Warn("Docker is not running")
+		log.Warn("!! Docker is not running")
 	}
 	log.Info("√ Docker is running")
 
 	//Check for installed docker-compose on client system
 	if err := exec.Command("docker-compose", "-v").Run(); err != nil {
-		log.Warn("Docker-Compose is not available")
+		log.Warn("!! Docker-Compose is not available")
```

Comment:
Should it be docker-compose ? or "Docker compose"?

Reply:
"Docker Compose" or "docker-compose" I guess

Reply:
Cool, I'll switch that!